klaytnetl/mappers/receipt_mapper.py

```diff
@@ -144,3 +152,13 @@ def receipt_to_dict(self, receipt: KlaytnRawReceipt) -> dict:
             "max_priority_fee_per_gas": receipt.max_priority_fee_per_gas,
             "max_fee_per_gas": receipt.max_fee_per_gas,
         }
+
+        if self.enrich and isinstance(receipt, KlaytnReceipt):
+            receipt_dict[
+                "block_unix_timestamp"
+            ] = receipt.block_timestamp.timestamp()
```

Comment:
Minor, but better to 
```suggestion
            receipt_dict["block_unix_timestamp"] = (
                receipt.block_timestamp.timestamp()
            )
```

Reply:
Gotcha. Thanks for the review.